File: src/Moryx.ControlSystem.Processes.Endpoints/Models/TracingModel.cs
// Copyright (c) 2026 Phoenix Contact GmbH & Co. KG
// Licensed under the Apache License, Version 2.0

using System.Runtime.Serialization;
using Moryx.Serialization;

namespace Moryx.ControlSystem.Processes.Endpoints
{
    [DataContract]
    public class TracingModel
    {
        [DataMember]
        public string Type { get; set; }

        [DataMember]
        public string Text { get; set; }

        [DataMember]
        public DateTime? Started { get; set; }

        [DataMember]
        public DateTime? Completed { get; set; }

        [DataMember]
        public int ErrorCode { get; set; }

        [DataMember]
        public Entry Properties { get; set; }
    }
}

File: src/Moryx.ControlSystem.ProcessEngine.Web/src/app/api/models/Moryx/ControlSystem/Processes/Endpoints/tracing-model.ts
/* tslint:disable */
/* eslint-disable */
import { Entry } from '@moryx/ngx-web-framework';
export interface TracingModel {
  completed?: string | null;
  errorCode?: number;
  properties?: Entry;
  started?: string | null;
  text?: string | null;
  type?: string | null;
}
